python/datasources/cdc_hiv.py

```diff
@@ -149,14 +189,50 @@ def load_df_from_data_dir(geo_level: str, breakdown: str):
     """
     load_df_from_data_dir fetches the csv file from the data dir
```

Comment:
maybe indicate here (and in the fn name?) that this fn actually loads several source tables and combines them into a single df for the requested breakdown

Reply:
or, would it make more sense to keep this function only loading a single CDC table, and then do the merging outside of the function? It looks like there is a lot of duplication between the multiple `read_csv` calls so could make sense to have the fn do a single read and then iterate of the needed types of file reads outside the fn ?

Reply:
Yes I agree, try to make this a general function that takes in a generic Atlas plus source.